src/support_sphere/lib/main.dart

```diff
@@ -25,8 +26,15 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return RepositoryProvider(
-      create: (_) => AuthenticationRepository(),
+    return MultiRepositoryProvider(
+      providers: [
+        RepositoryProvider(
+          create: (_) => AuthenticationRepository(),
+        ),
+        RepositoryProvider(
+          create: (context) => UserRepository(),
```

Comment:
Less of a "this should change" question and more of a "I don't know how Dart works" question:

Why does this Repository need `create: (context)` while others have `create: (_)`?

Reply:
Great catch! We are not using the context here so probably should be dropped.